mon-pix/tests/unit/models/modules/correction-response_test.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Model | Module | CorrectionResponse', function (hooks) {
+  setupTest(hooks);
+
+  test('CorrectionResponse model should exist with the right properties', function (assert) {
+    // given
+    const feedback = 'Good job!';
+    const status = 'ok';
+    const solution = 'a1b2c3d4-e5f6-7g8h-9i0j-k1l2m3n4o5p6';
```

Comment:
Pour être raccord avec le model 
```suggestion
      solutionId: 'a1b2c3d4-e5f6-7g8h-9i0j-k1l2m3n4o5p6';
```